packages/client/hmi-client/src/components/widgets/tera-slider.vue

```diff
@@ -60,6 +60,9 @@ const sidePanelTabStyle = computed(
 </script>
 
 <style scoped>
+aside {
+	position: relative;
+}
```

Comment:
Have we checked if there are any adverse impact of this change? tera-slider is used in tera-slider-panel, which is used across several places in the application.

Reply:
Yup it should all be working as normal - I made a small change to the `tera-preview-panel` since it was breaking there, but it should all be good now.